foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_9_0/UpdateDataAccessThriftServerEndpointMigration.java

```diff
@@ -0,0 +1,41 @@
+package com.boozallen.aissemble.upgrade.migration.v1_9_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractAissembleMigration;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+public class UpdateDataAccessThriftServerEndpointMigration extends AbstractAissembleMigration {
```

Comment:
Class level comment giving a high level overview of the migration

Reply:
Done